src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -152,9 +126,9 @@ describe('WorkspaceMenu - defined workspace (GCP or Azure)', () => {
     fireEvent.mouseOver(menuItem);
     // Assert
     if (canShare) {
-      expect(screen.queryByText(tooltipText.shareNoPermission)).toBeNull();
+      expect(screen.queryByRole('tooltip', { name: tooltipText.shareNoPermission })).toBeNull();
```

Comment:
Since we're no longer mocking tooltips, TooltipTrigger renders two elements containing the tooltip text: the visible tooltip itself, and a hidden element used to label TooltipTrigger's child. Thus, these queries would fail because they found multiple elements matching the text.

This narrows the query to look for only the tooltip itself, not the label element.
https://testing-library.com/docs/queries/about/#priority